sdk/cosmosdb/cosmos/tsconfig.strict.json

```diff
@@ -117,6 +117,7 @@
     "src/client/Database/index.ts",
     "src/documents/index.ts",
     "src/request/index.ts",
+    "src/request/DedicatedGatewayRequestOptions.ts",
```

Comment:
For my learning, why do we have this exclude list? And what's in `src/request/DedicatedGatewayRequestOptions.ts` that we need to have it here?

Reply:
ping offline for answer.